src/App.js

```diff
@@ -1,11 +1,14 @@
+import React from 'react';
+// import { useSelector } from 'react-redux';
```

Comment:
- [ ] Kindly remove this commented code line. If you need a previous version of the code you can refer to Git history.